docs/source/amp.rst

```diff
@@ -241,6 +241,12 @@ are safe to autocast.
 
 CPU Op-Specific Behavior
 ------------------------
+
+``bfloat16`` computation can be effectively conducted on platforms with AVX512 instruction set. 
+On platforms with AVX512 BFloat16 instruction, there will be an additional performance boost. 
+Query the instruction set via system ``lscpu | grep avx512_bf16`` or ``cat /proc/cpuinfo | grep avx512_bf16``. 
+In some cases ``float32`` (``float``) computation may be faster ``bfloat16`` one.
+
```

Comment:
Thanks for adding the comments that bf16 is slower than float on platforms without AVX512 BFloat16 instruction.
> ``bfloat16`` computation can be effectively conducted on platforms with AVX512 instruction set.

It may be confusing whether bf16 will be faster.

We may suggest to check if the platform support AVX512 BFloat16 instruction via ``lscpu | grep avx512_bf16`` or ``cat /proc/cpuinfo | grep avx512_bf16`` and not use bf16  on platforms without AVX512 BFloat16 instruction since bf16 is slower than float for computing bound ops like conv, linear and matmul on such platforms.
Could you please make your comments more clear ?